eva_cttv_pipeline/consequence_type.py

```diff
@@ -1,35 +1,33 @@
-
+from eva_cttv_pipeline import utilities
 
 __author__ = 'Javier Lopez: javild@gmail.com'
 
 
-def process_gene(consequence_type_dict, rs_id, ensembl_gene_id, so_term):
-    if rs_id in consequence_type_dict:
-        consequence_type_dict[rs_id].ensembl_gene_ids.add(ensembl_gene_id)
-        consequence_type_dict[rs_id].add_so_term(so_term)
+def process_gene(consequence_type_dict, var_id, ensembl_gene_id, so_term):
```

Comment:
Better to use 'variant' than simply 'var' (which could also stand for 'variable'!).